docs/statusbar.md

```diff
@@ -263,7 +263,7 @@ If the transition between status bar property changes should be animated. Suppor
 
 ### `backgroundColor` <div class="label android">Android</div>
 
-The background color of the status bar.
+The background color of the status bar. This was deprecated in API level 35.
```

Comment:
I think this should be:

1. Worded differently. Probably mention which version of RN is deprecating this prop
2. Move it inside a `:::info` or `:::warning` box

Reply:
It is not tied to RN version. The Android API backing this will have no effect from targetSdk 35 due to forced edge-to-edge.